src/postamble_minimal.js

```diff
@@ -76,6 +76,13 @@ function initRuntime(asm) {
   _emscripten_register_main_browser_thread_id(PThread.mainThreadBlock);
 #endif
 
+#if USE_WASM_WORKERS
+  if (ENVIRONMENT_IS_WASM_WORKER) {
+    __wasm_worker_initializeRuntime();
+    return;
+  }
```

Comment:
* We need to adjust the startup runtime .js to behave differently in Workers compared to main thread: it should not initialize the global data ctors etc. that are only the role of the main thread.